comp/core/tagger/component.go

```diff
@@ -39,13 +39,13 @@ type Component interface {
 	ReplayTagger() ReplayTagger
 	GetTaggerTelemetryStore() *telemetry.Store
 	Tag(entityID string, cardinality types.TagCardinality) ([]string, error)
```

Comment:
I didn't update the signature of the `Tag` method to avoid breaking rtloader. 

In a separate PR, I will add a new method to the tagger component (e.g. `TagWithEntityID`) which receives `entityID` as a struct, and will update calls to the tagger to use the new method, without breaking python integrations.